src/main/scala/com/gd/twitterstreamingtToDatalake/SparkSessionWrapper.scala

```diff
@@ -0,0 +1,14 @@
+package com.gd.twitterstreamingtToDatalake
+
+import org.apache.spark.SparkConf
+import org.apache.spark.streaming.{Seconds, StreamingContext}
+
+trait SparkSessionWrapper {
+
+  //Set the Spark Streaming Context
+  val conf = new SparkConf()
+  conf.setAppName("twitterStreaming").setMaster("local")
+
+  val ssc = new StreamingContext(conf, Seconds(5))
```

Comment:
`master` and `batch interval` should be parameterized via cmd arguments.
but it's not a high priority at the moment, so adding a TODO is fine.